ci/lint_python/subprocess_capture_checker.py

```
@@ -62,12 +80,12 @@
 DEFAULT_BASELINE = Path(__file__).resolve().parent / "subprocess_baseline.txt"
 
 # Regex pre-filter: quickly skip files with no subprocess usage at all.
-_SUBPROCESS_RE = re.compile(r"subprocess\s*\.")
+_SUBPROCESS_RE = re.compile(r"subprocess\s*\.|os\s*\.\s*(?:system|popen)\s*\(")
```

Ruff (0.16.0)

[error] Invalid unescaped character backspace, use "\b" instead

(PLE2510)